cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -230,6 +230,30 @@ pub type ERC20Transactor = assets_common::ERC20Transactor<
 	ERC20TransfersCheckingAccount,
 >;
 
+
+parameter_types! {
+	pub const ERC721TransferGasLimit: Weight = Weight::from_parts(1_000_000_000, 200_000);
+	pub const ERC721TransferStorageDepositLimit: Balance = 25_000_000_000;
+	pub ERC721TransfersCheckingAccount: AccountId = PalletId(*b"py/revch").into_account_truncating();
```

Comment:
We could probably use the same checking account as for ERC20s, since it's just an account that's used in the contract for holding the tokens